tests/snappi_tests/multidut/pfc/files/m2o_oversubscribe_lossy_helper.py

```diff
@@ -0,0 +1,432 @@
+            for val in prio_dscp_map[fp]:
+                flow_prio_dscp_list.append(val)
+        ipv4.priority.dscp.phb.values = [
+            ipv4.priority.dscp.phb.AF11,
+        ]
+        ipv4.priority.dscp.phb.values = flow_prio_dscp_list
+    elif 'Test Flow 1 -> 0' in flow.name:
+        ipv4.priority.dscp.phb.values = [
+            ipv4.priority.dscp.phb.CS1,
+        ]
+    elif 'Test Flow 2 -> 0' in flow.name:
+        ipv4.priority.dscp.phb.values = [
+            ipv4.priority.dscp.phb.AF11,
+        ]
+        ipv4.priority.dscp.phb.values = [
+            60, 61, 62, 63, 24, 25, 26, 27, 21, 23, 28, 29,
+            0, 2, 6, 59, 11, 13, 15, 58, 17, 16, 19, 54, 57,
+            56, 51, 50, 53, 52, 59, 49, 47, 44, 45, 42, 43, 40, 41
+        ]
+    else:
+        pass
+
+    ipv4.priority.dscp.ecn.value = ipv4.priority.dscp.ecn.CAPABLE_TRANSPORT_1
+    flow.size.fixed = data_pkt_size
+    flow.rate.percentage = flow_rate_percent
+    flow.duration.choice = flow.duration.CONTINUOUS
+    flow.duration.continuous.delay.nanoseconds = 0
+
+    flow.metrics.enable = True
+    flow.metrics.loss = True
+
+
+def start_traffic(api, flow_names):
+    logger.info("Starting traffic on :{}".format(flow_names))
+    ts = api.transmit_state()
+    ts.flow_names = flow_names
+    ts.state = ts.START
+    api.set_transmit_state(ts)
+
+
+def stop_traffic(api, flow_names):
+    logger.info("Stopping traffic on :{}".format(flow_names))
+    ts = api.transmit_state()
+    ts.flow_names = flow_names
+    ts.state = ts.STOP
+    api.set_transmit_state(ts)
+
+
+def __run_traffic(api, config, all_flow_names, exp_dur_sec, duthost):
+    """
+    Run traffic and dump per-flow statistics
+
+    Args:
+        api (obj): SNAPPI session
+        config (obj): experiment config (testbed config + flow config)
+        all_flow_names (list): list of names of all the flows
+        exp_dur_sec (int): experiment duration in second
+
+    Returns:
+        per-flow statistics (list)
+    """
+    test_traffic, bg_traffic = [], []
+    api.set_config(config)
+
+    logger.info('Wait for Arp to Resolve ...')
+    wait_for_arp(api, max_attempts=10, poll_interval_sec=2)
+    duthost.command("sonic-clear counters \n")
+    logger.info('Starting transmit on all flows ...')
+    for flow in all_flow_names:
+        if 'Test Flow' in flow:
+            test_traffic.append(flow)
+        else:
+            bg_traffic.append(flow)
+    logger.info('Start Traffic..')
+    start_traffic(api, bg_traffic)
+    time.sleep(5)
+    start_traffic(api, test_traffic)
+
+    logger.info('Stop Traffic..')
+    stop_traffic(api, all_flow_names)
+    var = duthost.shell("show interface counters")['stdout']
+    """ Dump per-flow statistics """
+    request = api.metrics_request()
+    request.flow.flow_names = all_flow_names
+    rows = api.get_metrics(request).flow_metrics
+    file1 = open('myfile.txt', 'w+')
+    file1.writelines(var)
+    return rows
+
+
+def __verify_results(rows,
```

Comment:
add this to `traffic_generator.py`?

Reply:
added verification to traffic_generator.py